apps/glusterfs/errors.go

```diff
@@ -25,6 +25,7 @@ var (
 	ErrKeyExists        = errors.New("Key already exists in the database")
 	ErrNoReplacement    = errors.New("No Replacement was found for resource requested to be removed")
 	ErrCloneBlockVol    = errors.New("Cloning of block hosting volumes is not supported")
+	ErrInvalidRequest   = errors.New("Invalid request")
```

Comment:
John, I had looked at the existing errors and none of them looks like a match to give a better hint in this case, if you have any thoughts you can share here, so that I will drop it.

This description I had copied from errno.h
```
EINVAL          Invalid argument (POSIX.1-2001).
```

which looked more generic, I can tune it specific to my functionality of code, but the reusability will be compromised.

Let me know what you think.